mllm/backends/qnn/aot/passes/PTQPass.cpp

```
@@ -148,6 +148,49 @@ void _recursiveSolveNormalImpl(const ir::Val::ptr_t& v) {
       auto this_spec = std::static_pointer_cast<ir::linalg::QuantizationSpecAsymPerTensor>(f_spec->spec_);
       this_spec->scale = scale;
       this_spec->zero_point = zero_point;
+      auto min_v = this_spec->quant_min;
+      auto max_v = this_spec->quant_max;
+
+      // Check if this tensor is constant tensor. Then we need to quantize it.
+      double constant_v = 0;
+      DataTypes constant_dtype = kFloat32;
```

Clang (14.0.6)

[error] variable 'constant_dtype' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)